src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -60,6 +137,93 @@ public void isDueOn_monthlyFiltered() {
     }
 
     @Test
+    public void upcomingDueDates_dailySimple() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)